README.md

```diff
@@ -1,2 +1,17 @@
 # comp110-worksheet-B
 Base repository for COMP110 worksheet B
+Create a list of random words which the same length
```

Comment:
The worksheet asked you to provide pseudocode for the algorithm to compute the similarity between the guessed word and the password, which you have not done. Be sure to read the specification carefully in future!